contrib/alos2proc_f/src/cbind.f90

```diff
@@ -28,7 +28,8 @@ subroutine c_rect(infile,outfile,ndac,nddn,nrac,nrdn,a,b,c,d,e,f,filetype,intsty
         call rect(infile,outfile,ndac,nddn,nrac,nrdn,a,b,c,d,e,f,filetype,intstyle)
     end subroutine
 
-    subroutine c_rect_with_looks(infile,outfile,ndac,nddn,nrac,nrdn,a,b,c,d,e,f,lac,ldn,lac0,ldn0,filetype,intstyle) bind(c, name="c_rect_with_looks")
+    subroutine c_rect_with_looks(infile,outfile,ndac,nddn,nrac,nrdn,a,b,c,d,e,f,lac,ldn,lac0,ldn0,filetype,intstyle) &
+            bind(c, name="c_rect_with_looks")
```

Comment:
Had to wrap this line to avoid fortran -Werror=truncation. For some reason -ffixed-line-length-none did not prevent this.